### PR TITLE
Removing Expects from Lading

### PR DESCRIPTION
**Pull request by dmalis18**
### What does this PR do?

Began the process of converting expects to the question mark operator

### Motivation

What inspired you to submit this pull request?

### Related issues

SMPTNG-114

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: integration/ducks/src/main.rs [hunk 1]
```diff
@@ -335,9 +339,7 @@ async fn main() -> Result<(), anyhow::Error> {
     debug!("hello from ducks");
 
     // Every ducks-sheepdog pair is connected by a unique socket file
-    let ducks_comm_file = std::env::args()
-        .nth(1)
-        .expect("ducks socket file argument missing");
+    let ducks_comm_file = std::env::args().nth(1).ok_or(anyhow::anyhow!("Error"))?;
```
Comment (GeorgeHahn) [inline integration/ducks/src/main.rs:342]: The old expect string would be a good fit for this error string

File: lading/src/bin/lading.rs [hunk 2]
```diff
@@ -37,6 +37,12 @@ use tracing_subscriber::{fmt::format::FmtSpan, util::SubscriberInitExt};
 enum Error {
     #[error("Target related error: {0}")]
     Target(target::Error),
+
+    #[error("Set recorder error: {0}")]
+    SetRecorder(#[from] SetRecorderError),
```
Comment (GeorgeHahn) [inline lading/src/bin/lading.rs:42]: Is this variant used? It looks like it might not be.

File: lading/src/captures.rs [hunk 3]
```diff
@@ -17,13 +17,22 @@ use std::{
...
 use tracing::{debug, info, warn};
 use uuid::Uuid;
 
 use crate::signals::Phase;
 
+/// Errors produced by [`CaptureManager`]
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    /// Wrapper around [`SetRecorderError`].
+    #[error("Set recorder error: {0}")]
```
Comment (GeorgeHahn) [inline lading/src/captures.rs:32]: This error text could be more descriptive - how about something like 'failed to set recorder: {0}'?

File: lading/src/captures.rs [hunk 4]
```diff
@@ -160,13 +171,15 @@ impl CaptureManager {
     /// This function only exits once a shutdown signal is received.
     ///
     /// # Panics
+    /// # Errors
     ///
+    /// Will return `Err` if there is already a global recorder set
     /// None known.
```
Comment (GeorgeHahn) [inline lading/src/captures.rs:177]: It looks like these '# Panics' and '# Errors' sections are mixed together.

**Review by goxberry — APPROVED** [review]
LGTM with the recent changes.

**Review by GeorgeHahn — APPROVED** [review]

**Pull request merged**